lib/debounce.js

```diff
@@ -1,16 +1,11 @@
-import requestAnimationFrame from './requestAnimationFrame'
+export default function debounce (func, time = 300) {
+  let timeoutId = null
 
-export default function debounce (fn) {
-  let waiting = false
-
-  return function (...args) {
-    if (!waiting) {
-      waiting = true
-
-      requestAnimationFrame(() => {
-        fn(...args)
-        waiting = false
-      })
+  return function () {
```

Comment:
Gotcha. We have an application that runs in an iframe that gets resized via an animation. This animation is triggered whenever the Radio selector changes in size (selecting different options); the previous `debounce` would trigger a lot of costly `Collapsible.updateHeight` calls, that is why I changed it.

Should we keep the previous implementation, but rename it to something else (since it is not doing what is expected of a `debounce` function) ? And also keep the new implementation to be used in scenarios such as the one I've described above?